cf-java-logging-support-core/src/test/java/com/sap/hcp/cf/logging/common/EnvMap.java

```diff
@@ -11,35 +11,8 @@ public class EnvMap {
-                                                  "\"cf_api\":\"https://api.cf.sap.hana.ondemand.com\", " +
-                                                  "\"host\":\"0.0.0.0\", " +
-                                                  "\"instance_id\":\"34b12c70-5443-426f-75c4-3260\", " +
-                                                  "\"instance_index\":\"" + VCAP_INSTANCE_IDX + "\", " +
-                                                  "\"limits\":{  " +
-                                                  "\"disk\":1024, " +
-                                                  "\"fds\":16384, " +
-                                                  "\"mem\":1024 " +
-                                                  "}, " +
-                                                  "\"name\":\"d048888-logging-sample-app\", " +
-                                                  "\"organization_id\":\"" + VCAP_ORGANIZATION_ID + "\", " +
-                                                  "\"organization_name\":\"" + VCAP_ORGANIZATION_NAME + "\", " +
-                                                  "\"port\":8080, " +
-                                                  "\"process_id\":\"a6026141-8d69-4e9d-b62d-8e1d982f7953\", " +
-                                                  "\"process_type\":\"web\", " +
-                                                  "\"space_id\":\"" + VCAP_SPACE_ID + "\", " +
-                                                  "\"space_name\":\"" +VCAP_SPACE_NAME + "\", " +
-                                                  "\"uris\":[  " +
-                                                  "\"testapp.testdomain.com\" " +
-                                                  "], " +
-                                                  "\"version\":\"92720bd3-2735-489d-bf4b-e5bb05c752e7\" " +
-                                                  "}";
+    public static final String VCAP_APPLICATION =
+            "{" + "\"application_id\":\"" + VCAP_APP_ID + "\", " + "\"application_name\":\"" + VCAP_APP_NAME + "\", " + "\"application_uris\":[  " + "\"testapp.testdomain.com\" " + "], " + "\"application_version\":\"92720bd3-2735-489d-bf4b-e5bb05c752e7\", " + "\"cf_api\":\"https://api.cf.sap.hana.ondemand.com\", " + "\"host\":\"0.0.0.0\", " + "\"instance_id\":\"34b12c70-5443-426f-75c4-3260\", " + "\"instance_index\":\"" + VCAP_INSTANCE_IDX + "\", " + "\"limits\":{  " + "\"disk\":1024, " + "\"fds\":16384, " + "\"mem\":1024 " + "}, " + "\"name\":\"d048888-logging-sample-app\", " + "\"organization_id\":\"" + VCAP_ORGANIZATION_ID + "\", " + "\"organization_name\":\"" + VCAP_ORGANIZATION_NAME + "\", " + "\"port\":8080, " + "\"process_id\":\"a6026141-8d69-4e9d-b62d-8e1d982f7953\", " + "\"process_type\":\"web\", " + "\"space_id\":\"" + VCAP_SPACE_ID + "\", " + "\"space_name\":\"" + VCAP_SPACE_NAME + "\", " + "\"uris\":[  " + "\"testapp.testdomain.com\" " + "], " + "\"version\":\"92720bd3-2735-489d-bf4b-e5bb05c752e7\" " + "}";
 
```

Comment:
what is the benefit of "onelining" all this fields?

Reply:
No benefit. Automatic code-formatting.